libs/application/template-api-modules/src/lib/modules/templates/driving-license-book-update-instructor/driving-license-book-update-instructor.service.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from '@nestjs/common'
+import { TemplateApiModuleActionProps } from '../../../types'
+import { DrivingLicenseBookService } from '@island.is/api/domains/driving-license-book'
```

Comment:
We dont want the application system to depend on the api.  You should communicate with the  client library directly  instead.